data/scripts.js

```diff
@@ -660,8 +660,8 @@ let BattleScripts = {
 		}
 
 		if (target) {
-			/**@type {?boolean | number} */
-			let didSomething = false;
+			/**@type {?boolean | number | undefined} */
+			let didSomething;
```

Comment:
I would explicitly set it to `undefined` here, then.

Reply:
I tried that, but doing so gives a warning that tells me explicitly _not_ to do that.

Reply:
Let's just get rid of that warning.